tests/components/alert/test_init.py

```diff
@@ -223,6 +223,42 @@ def record_event(event):
     assert len(events) == 2
 
 
+async def test_no_notifiers(hass: HomeAssistant) -> None:
+    """Test we send no notifications when there are not no."""
```

Comment:
What's this actually saying? :-)

Reply:
Uh... that I should not have made such a long day 😃